content/02.introduction.md

```diff
@@ -19,19 +20,19 @@ Therefore, advanced correlation coefficients could immediately find wide applica
 
 
 The Pearson and Spearman correlation coefficients are widely used because they reveal intuitive relationships and can be computed quickly.
-However, they are designed to capture linear or monotonic patterns (referred to as linear-only) and may miss complex yet critical relationships.
-Novel coefficients have been proposed as metrics that capture nonlinear patterns such as the Maximal Information Coefficient (MIC) [@pmid:22174245] and the Distance Correlation (DC) [@doi:10.1214/009053607000000505].
-MIC, in particular, is one of the most commonly used statistics to capture more complex relationships, with successful applications across several domains [@pmid:33972855; @pmid:33001806; @pmid:27006077].
-However, the computational complexity makes them impractical for even moderately sized datasets [@pmid:33972855; @pmid:27333001].
-Recent implementations of MIC, for example, take several seconds to compute on a single variable pair across a few thousand objects or conditions [@pmid:33972855].
-We previously developed a clustering method for highly diverse datasets that significantly outperformed approaches based on Pearson, Spearman, DC and MIC in detecting clusters of simulated linear and nonlinear relationships with varying noise levels [@doi:10.1093/bioinformatics/bty899].
```

Comment:
Does the tool have a problem where it removes citations?

Reply:
We found that this model (`text-davinci-edit-001`) has many problems. I recommend taking a look at the results using [`text-davinci-003`](https://github.com/greenelab/ccc-manuscript/pull/28).